packages/info-sprinkle/src/InfoSprinkle/InfoSprinkle.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { cx } from '@leafygreen-ui/emotion';
+import Icon from '@leafygreen-ui/icon';
+import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+import Tooltip from '@leafygreen-ui/tooltip';
+
+import { iconBaseStyles, iconThemeStyles } from './InfoSprinkle.styles';
+import { Align, InfoSprinkleProps, Justify } from './InfoSprinkle.types';
+
+export const InfoSprinkle = React.forwardRef<HTMLDivElement, InfoSprinkleProps>(
```

Comment:
forwardRef should accept `HTMLSpanElement` right? Since we're spreading the ref on a `<span />`